scripts/get_dues.py

```diff
@@ -67,7 +67,7 @@
     for row in records:
       outcsv.writerow([row[x] for x in column_names])
 
-    print("Successfully output dues to {}".format(CSV_FILE))
+    print(("Successfully output dues to {}".format(CSV_FILE)))
```

Comment:
No. I did not pass -p because many print statements in the database are not function calls. By the docs:

>When the -p is passed, 2to3 treats print as a function instead of a statement. This is useful when `from __future__ import print_function` is being used. If this option is not given, the print fixer will surround print calls in an extra set of parentheses because it cannot differentiate between the print statement with parentheses (such as `print ("a" + "b" + "c")`) and a true function call.

Nevertheless, it shall not matter, so I don't choose to exclude it from the commit. This PR is about py3 conversion after all -- style is a big issue that needs to be addressed elsewhere...